python/cuml/benchmark/algorithms.py

```diff
@@ -221,7 +236,7 @@ def all_algorithms():
             cuml_args={},
             name="NearestNeighbors",
             accepts_labels=False,
-            bench_func=fit_kneighbors
+            bench_func=fit_kneighbors,
```

Comment:
Is this just a convenience for adding new lines in the future without having to think about adding the commas?